src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -134,6 +135,16 @@ public function validate($entity, Constraint $constraint)
         $errorPath = null !== $constraint->errorPath ? $constraint->errorPath : $fields[0];
         $invalidValue = isset($criteria[$errorPath]) ? $criteria[$errorPath] : $criteria[$fields[0]];
 
+        $vars = array();
+
+        if (preg_match_all('/{{ ([a-zA-Z0-9_]+) }}/i', $constraint->message, $vars) > 0) {
+            $accessor = PropertyAccess::createPropertyAccessor();
+
+            foreach ($vars[1] as $var) {
+                $constraint->message = str_replace(sprintf('{{ %s }}', $var), $accessor->getValue($entity, $var), $constraint->message);
```

Comment:
please don't make replacements yourselves. It makes the message incompatible with usage of the translator. Replacements must be applied after the translator is used